chainforge/react-server/src/backend/utils.ts

```diff
@@ -1927,7 +1927,8 @@ export const toStandardResponseFormat = (r: Dict | string) => {
   const resp_obj: LLMResponse = {
     vars: r?.fill_history ?? {},
     metavars: r?.metavars ?? {},
-    uid: r?.uid ?? r?.batch_id ?? uuid(),
+    // uid: r?.uid ?? r?.batch_id ?? uuid(),
+    uid: uuid(), // uuid should be unique when it was generated at the backend. For new, regenerate here to make it unique.
```

Comment:
Why was this change made? This changes a core function of the ChainForge codebase.